appcheck/firebase-appcheck-safetynet/gradle.properties

```diff
@@ -1,2 +1,2 @@
-version=16.1.3
+version=17.0.0
```

Comment:
Safetynet is deprecated and will stop working on June 2024

https://firebase.google.com/docs/app-check/android/safetynet-provider

feel free to ignore it